aleph-client/src/contract/event.rs

```diff
@@ -41,14 +41,39 @@
 //! #   Ok(())
 //! # }
 //! ```
+//!
+//! To use the fetching feature you will need to pass a connection, transaction coordinates and
+//! a corresponding contract to the [get_contract_events] function.
+//!
+//! ```no_run
+//! # use aleph_client::{AccountId, Connection, SignedConnection};
+//! # use aleph_client::contract::ContractInstance;
+//! # use aleph_client::contract::event::{get_contract_events, listen_contract_events};
+//! # use anyhow::Result;
+//! use futures::{channel::mpsc::unbounded, StreamExt};
+//!
+//! # async fn example(conn: Connection, signed_conn: SignedConnection, address: AccountId, path: &str) -> Result<()> {
+//! let contract = ContractInstance::new(address, path)?;
+//!
+//! let tx_info = contract.contract_exec0(&signed_conn, "some_method").await?;
+//!
+//! println!("Received events {:?}", get_contract_events(&conn, &contract, tx_info).await);
+//!
+//! #   Ok(())
+//! # }
+//! ```
```

Comment:
Suggestion: not that there is these two ways of getting events, I propose:

1. Moving the docs for the two ways to the docstrings of the two functions.
2. Changing this docstring to focus on a comparison between the two methods.

Reply:
done